package.json

```diff
@@ -29,10 +29,18 @@
   "bugs": {
     "url": "https://github.com/nherment/seneca-github-auth/issues"
   },
+  "scripts": {
+    "lint": "lab test -dL",
+    "annotate": "docco github-auth.js -o doc"
+  },
   "homepage": "https://github.com/nherment/seneca-github-auth",
   "dependencies": {
-    "passport-github": "0.1.5",
-    "lodash": "~3.10.0"
+    "passport-github": "1.1.0",
+    "lodash": "~4.2.1",
+    "docco": "0.7.0",
```

Comment:
also you need all these three plugins installed: 
eslint-config-seneca
eslint-plugin-standard
eslint-plugin-hapi

after you install them, please run the linter again, you might find more linting issues